tensorflow/python/ops/numpy_ops/np_arrays_test.py

```diff
@@ -69,6 +82,60 @@ def testNeg(self):
     a = t2a(ops.convert_to_tensor(value=[1.0, 2.0]))
     self.assertAllEqual([-1.0, -2.0], -a)
 
+  def testSum(self):
+    for d in self.data:
+      np_array = np.array(d)
+      test_array = np_array_ops.array(d)
+      self.assertAllClose(np_array.sum(), test_array.sum())
```

Comment:
Instead of this, is it possible to test using the _testBinOp below? It'll help expand coverage of these for all dtypes.